.envrc

```diff
@@ -0,0 +1,3 @@
+export GITLAB_KEY=your-key
```

Comment:
It might be easier to do this with the `dotenv` gem

Reply:
yea, I just pushed this since I use `direnv` everywhere 🙂